### PR TITLE
Update continuous-integration-faqs.md

### PR DESCRIPTION
**Pull request by austinthao5**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
Thanks for your contribution @austinthao5 
Could you please sign the [CLA](https://cla-assistant.io/harness/developer-hub?pullRequest=7389)? 

https://github.com/harness/developer-hub/pull/7389#issuecomment-2231723827

thanks.

**Comment by austinthao5** [comment]
CLA has been signed. Let me know if there's any other issues

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 1]
```diff
@@ -28,6 +28,34 @@ Yes, each stage can have a different build infrastructure. Additionally, dependi
 
 No. Your build infrastructure can be configured to use whichever tools you like. For example, Harness Cloud build infrastructure includes pre-installed versions of xcode and other tools, and you can install other tools or versions of tools that you prefer to use. For more information, go to the [CI macOS and iOS development guide](https://developer.harness.io/docs/continuous-integration/development-guides/ci-ios).
 
+### What's the difference between `CI_MOUNT_VOLUMES`, `ADDITIONAL_CERTS_PATH`, and `DESTINATION_CA_PATH`?
```
Comment (dewan-ahmed) [inline kb/continuous-integration/continuous-integration-faqs.md:31]:
```suggestion
### What's the difference between CI_MOUNT_VOLUMES, ADDITIONAL_CERTS_PATH, and DESTINATION_CA_PATH?
```

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 2]
```diff
@@ -28,6 +28,34 @@ Yes, each stage can have a different build infrastructure. Additionally, dependi
...
+  value: "/tmp/ca.bundle"
+```
+
+`DESTINATION_CA_PATH` - An environment variable used for CI Build Stages. This variable should be set to a comma-separated list of files where the certificate should be mounted. For example,
+```
+- name: DESTINATION_CA_PATH
+  value: "/etc/ssl/certs/ca-bundle.crt,/kaniko/ssl/certs/additional-ca-cert-bundle.crt"
+```
+
+`ADDTIONAL_CERTS_PATH` and `CI_MOUNT_VOLUMES` work in tandem to ensure certificates are mounted on the Kubernetes Build Infrastructure where as `DESTINATION_CA_PATH` does not require other environment variables to mount certificates. Instead, `DESTINATION_CA_PATH` relies on the certificate being mounted at `/opt/harness-delegate/ca-bundle` in order to copy the certificate to the provided comma-separated list of file paths.
```
Comment (dewan-ahmed) [inline kb/continuous-integration/continuous-integration-faqs.md:52]:
```suggestion
`ADDTIONAL_CERTS_PATH` and `CI_MOUNT_VOLUMES` work in tandem to ensure certificates are mounted on the Kubernetes Build Infrastructure, whereas `DESTINATION_CA_PATH` does not require other environment variables to mount certificates. Instead, `DESTINATION_CA_PATH` relies on the certificate being mounted at `/opt/harness-delegate/ca-bundle` in order to copy the certificate to the provided comma-separated list of file paths.
```

**Review by dewan-ahmed — CHANGES_REQUESTED** [review]
Screenshot 2024-08-01 at 11 53 13 PM
Made some formatting changes due to how the backtick enclosed text was appearing.

**Review by dewan-ahmed — APPROVED** [review]
Changes done via https://github.com/harness/developer-hub/pull/7389/commits/ea2643ce57190d65fb2319116cbdf493d34571d1 and and https://github.com/harness/developer-hub/pull/7389/commits/9705f78949b098cef993953082cf217af62d6f00

**Pull request merged**